### PR TITLE
model: fix find_all for nit/next

### PR DESCRIPTION
**Pull request by Morriar**
Signed-off-by: Alexandre Terrasa alexandre@moz-code.org

### CODE REVIEW

File: src/model/notifications.nit [hunk 1]
```diff
@@ -56,7 +56,7 @@ end
 class PlayerNotificationRepo
 	super MongoRepository[PlayerNotification]
 
-	redef fun find_all(q) do
+	redef fun find_all(q, s, l) do
```
Comment (privat) [inline src/model/notifications.nit:59]: no sql?

**Comment by privat** [comment]
still issues :(

**Comment by Morriar** [comment]
tes this please

**Comment by Morriar** [comment]
test this please

**Pull request merged**